core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -11,6 +12,30 @@ class ReligionManager {
 
     var pantheonBelief: String? = null
     
+    var greatProphetsEarned = 0
+
+    fun clone(): ReligionManager {
+        val clone = ReligionManager()
+        clone.pantheonBelief = pantheonBelief
+        clone.storedFaith = storedFaith
+        return clone
+    }
+
+    fun startTurn() {
+        if (canGenerateProphet() && storedFaith > faithForNextGreatProphet()) {
+            val prophetSpawnChange = (5f + storedFaith - faithForNextGreatProphet()) / 100f
+            if (Random.nextFloat() < prophetSpawnChange) {
```

Comment:
Random.nextFloat should be seeded by the turn number, to minimize 'retry and hope RNGesus smiles'